README.md

```diff
@@ -80,6 +80,25 @@ operate. Using the data model, serializers and deserializers:
     }
     ```
 
+### Zigmod
+
+1. [Read this tutorial](https://nektro.github.io/zigmod/tutorial.html) on how to setup a new zigmod project.
+
+    Note: getty is not hosted on aquila.red ([see this comment for details](https://github.com/nektro/zigmod/issues/63#issuecomment-1063205053)).
+
+1. Add getty to zigmod.yml
+
+    ```
+    root_dependencies:
```

Comment:
I think this should be `dependencies`.

IIUC, using `root_dependencies` would mean that if a user pulls in a library called `foobar` (via Zigmod) and `foobar` depends on Getty (via Zigmod), then Getty won't be pulled in by `foobar` since it isn't the root module.

```suggestion
    dependencies:
```